cds_ils/patrons/permissions.py

```diff
@@ -25,9 +25,17 @@ def retrieve_patron_loans_permission(*args, **kwargs):
     )
 
 
+def backoffice_permission(*args, **kwargs):
+    """Return permission to allow only librarians and admins."""
+    return Permission(backoffice_access_action)
```

Comment:
Hm, why do we need to redefine `backoffice_permission`? (there is one in ILS already)

Reply:
Yes, you are right this code is lefotver from a bad rebase, thanks!

Reply:
Okay! I've implemented the importer permissions in that file already but maybe it's not the best place. In any case this will be done in the other PR.